srcpkgs/ProTracker/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'ProTracker'
+pkgname=ProTracker
+version=0.20b
+revision=1
+hostmakedepends="subversion"
+makedepends="SDL2-devel"
+depends="SDL2"
```

Comment:
There is no need to depend on SDL2 it will be found by the shlibs tracking system.

Reply:
Ok, should i modify the template or is it good?